docker-compose.yml

```diff
@@ -37,7 +37,7 @@ services:
       - tool_config:/root/.config
       - static_volume:/usr/src/app/staticfiles/
     environment:
-      - DEBUG=0
```

Comment:
nooo ;p

Reply:
my bad, forgot to revert this back after troubleshooting.